cli/src/main/java/com/box/l10n/mojito/cli/command/RepositoryMachineTranslationCommand.java

```diff
@@ -0,0 +1,84 @@
+package com.box.l10n.mojito.cli.command;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.box.l10n.mojito.cli.command.param.Param;
+import com.box.l10n.mojito.cli.console.ConsoleWriter;
+import com.box.l10n.mojito.rest.client.RepositoryMachineTranslationClient;
+import com.box.l10n.mojito.rest.entity.PollableTask;
+import com.box.l10n.mojito.rest.entity.RepositoryMachineTranslationBody;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.fusesource.jansi.Ansi.Color;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+/**
+ * Command to machine translate untranslated strings in a repository.
+ *
+ * @author jaurambault
+ */
+@Component
+@Scope("prototype")
+@Parameters(
+    commandNames = {"repository-machine-translate"},
+    commandDescription = "Machine translate untranslated strings in a repository")
+public class RepositoryMachineTranslationCommand extends Command {
```

Comment:
Rather than needing to run this command to translate the repo every time - would it not be better to have it implemented as part of the pull command (as an additional option) - so that every time someone wants to see a fully translated app - they can pass in the extra parameter? 
In that case - importing the MT translations into the repo and saving them explicitly against the text units can be avoided.